### DISCUSSION TITLE
Video dubbing gradio

### DISCUSSION THREAD

**Discussion by sarpba** [SWivid/F5-TTS#410] [Show and tell]
Hi, I made a gradio for video dubbing. 
Sorry if it's a bit rudimentary, this is my first Gradio.

It's working under linux only at the moment.

If you finetuned your modell can try this. I used for english to hungarian, and it's working...
Put your .pt model and vocab txt into AI_sync/TTS/your_folder

https://github.com/sarpba/AI_sync

**Comment by sarpba** [comment]
Nobody cares? It turned out pretty good though. :)

**Reply by sngraam** [reply]
hii bro - you create a nice project anyways where are you from ?

**Reply by sarpba** [reply]
Thank. Hi, I'm from Hungary.

**Reply by sngraam** [reply]
are you free? I would like to discuss something important with you. Could you please share your email or any other contact information?

**Reply by sngraam** [reply]
check email

**Reply by sngraam** [reply]
Thank You
